gnucash/gtkbuilder/dialog-price.glade

```diff
@@ -9,16 +9,16 @@
     </columns>
     <data>
       <row>
-        <col id="0" translatable="yes">Bid</col>
+        <col id="0" translatable="yes" context="Price">Bid</col>
```

Comment:
We have started to distinguish a "Quote" –offered at some market– and a real "Price" –paid for an Item. So better use context="Quote" .

I will have to look, if the whole dialog should be renamed.